src/validation/DriftyValidation.java

```diff
@@ -0,0 +1,48 @@
+package validation;
+
+import singleton.CreateLogs;
+
+import static constants.DriftyConstants.*;
+
+import java.util.Scanner;
+
+
+public final class DriftyValidation {
+
+    public static final CreateLogs logger = CreateLogs.getInstance();
+    private static Scanner SC = new Scanner(System.in);
+
+    /**
+     * This method performs Yes-No validation and returns the boolean value accordingly.
+     *
+     * @param input        Input String to validate.
+     * @param printMessage The message to print to re-input the confirmation.
+     * @return true if the user enters Y [Yes] and false if not.
+     */
+    public static boolean yesNoValidation(String input, String printMessage) {
+
+        while (true) {
+            if (input.length() == 0) {
+                System.out.println("Please enter Y for yes and N for no!");
+                logger.log("ERROR", "Please enter Y for yes and N for no!");
```

Comment:
Add a constant in `DriftyConstants.java` file and make the below changes.
```suggestion
                logger.log(LOGGER_ERROR, ENTER_Y_OR_N);
```